README.md

```diff
@@ -41,9 +41,22 @@ dnf install libtool clang-devel bison cmake ninja-build samba flex texinfo glib2
 
 # Basic usage
 
-If you want to start up a QEMU VM running CheriBSD run `cheribuild.py run-riscv64-purecap -d` (-d means build all dependencies).
```

Comment:
I made this change so that the two commands given match stylistically in how they are presented from an English language point of view.  In the previous version we did not mention Morello and adding that caused me to reword this little section.

Reply:
Could just word the addition to match the existing style though? "If you would like the VM to use Arm's Morello ISA instead of CHERI-RISC-V use `cheribuild.py run-morello-purecap -d`."

Reply:
I find it too passive, but, sure we can make it match.